aws-datastore/src/test/java/com/amplifyframework/datastore/storage/InMemoryStorageAdapter.java

```diff
@@ -26,10 +26,14 @@
 import com.amplifyframework.core.model.ModelSchema;
 import com.amplifyframework.core.model.query.QueryOptions;
 import com.amplifyframework.core.model.query.predicate.QueryPredicate;
+import com.amplifyframework.core.model.query.predicate.QueryPredicates;
 import com.amplifyframework.datastore.DataStoreException;
 
+import com.google.common.base.Predicates;
```

Comment:
No `com.google.common`, please. We can use Gson, itself, since we have a dependency on it. But we ought not to use its transitive dependencies. This one might be from Guava, I guess?

Reply:
that shouldn't have made it here... must have been auto imported from when i accidentally auto-completed. I removed them though.